editors/neovim/Portfile

```
@@ -48,7 +48,12 @@ cmake.build_type        Release
 configure.args-append   -DLUA_PRG=${prefix}/bin/luajit
 
 # Building parsers is normally an extra step, see https://github.com/neovim/neovim/issues/29042
-patchfiles              embed-parsers-build.diff
+patchfiles              embed-parsers-build.diff  \
```

Extra space could be removed.

```suggestion
patchfiles              embed-parsers-build.diff \
```